gossip/src/crds.rs

```diff
@@ -669,6 +677,22 @@ impl CrdsDataStats {
                 self.votes.put(slot, num_nodes + 1);
             }
         }
+
+        if should_report_message_signature(&entry.value.signature) {
+            datapoint_info!(
+                "cluster_info_crds_message_signatures",
+                (
+                    "crds_origin",
+                    entry.value.pubkey().to_string().get(..8),
+                    Option<String>
+                ),
+                (
+                    "crds_signature",
```

Comment:
slight preference if we remove `crds_` prefix from `crds_origin` and `crds_signature`.
The metric metric already says `crds`.

Also, the metric name could be `gossip_crds_sample`.

Reply:
ok that makes sense. will update